templates/villageHQ.php

```diff
@@ -23,7 +23,7 @@
                 seatData: <?= json_encode(VillageApiPresenter::seatDataResponse($system, $player)) ?>,
                 pointsData: <?= json_encode(VillageApiPresenter::pointsDataResponse($system, $player)) ?>,
                 diplomacyData: <?= json_encode(VillageApiPresenter::diplomacyDataResponse($system, $player)) ?>,
-                resourceData: <?= json_encode(VillageApiPresenter::resourceDataResponse($system, $player, 7)) ?>,
+                resourceData: <?= json_encode(VillageApiPresenter::resourceDataResponse($system, $player, 1)) ?>,
```

Comment:
What does this number stand for - should we put it in a constant somewhere?

Reply:
That's just the number of days, we can make explicit when we call the function for clarity. `days: 7`